docusaurus/docs/how-to-guides/app-plugins/fetch-data-from-frontend-to-app.md

```diff
@@ -0,0 +1,132 @@
+---
+id: fetch-data-from-frontend-to-app
+title: Fetch data from frontend code to app plugin using the data proxy
```

Comment:
This title is a bit confusing to read for me, it makes it looks like the frontend code and the app plugin are two different things that interact with each other.

Reply:
Maybe change it to the following?

```suggestion
title: Fetch data from frontend code in app plugin using the data proxy
```